go/vt/sqlparser/sql.y

```diff
@@ -1575,9 +1567,17 @@ show_statement:
   {
     $$ = &Show{Type: string($2)}
   }
-| SHOW VSCHEMA_TABLES
```

Comment:
Should we remove this token now from the grammar? It does not look like we need `VSCHEMA_TABLES` anymore